src/lib/button-toggle/button-toggle.spec.ts

```diff
@@ -607,66 +608,77 @@ describe('MatButtonToggle without forms', () => {
     }));
 
     it('should focus on underlying input element when focus() is called', () => {
-      let nativeRadioInput = buttonToggleDebugElement.query(By.css('input')).nativeElement;
-      expect(document.activeElement).not.toBe(nativeRadioInput);
+      let nativeButton = buttonToggleDebugElement.query(By.css('button')).nativeElement;
+      expect(document.activeElement).not.toBe(nativeButton);
 
       buttonToggleInstance.focus();
       fixture.detectChanges();
 
-      expect(document.activeElement).toBe(nativeRadioInput);
+      expect(document.activeElement).toBe(nativeButton);
     });
 
     it('should not assign a name to the underlying input if one is not passed in', () => {
-      expect(buttonToggleNativeElement.querySelector('input')!.getAttribute('name')).toBeFalsy();
+      expect(buttonToggleNativeElement.querySelector('button')!.getAttribute('name')).toBeFalsy();
     });
 
+    it('should have correct aria-pressed attribute', () => {
+      expect(buttonToggleNativeElement.querySelector('button')!.getAttribute('aria-pressed'))
```

Comment:
You can move out the `buttonToggleNativeElement.querySelector('button')` into a variable since it'll always point to the same element.